sig/datadog/opentelemetry/sdk/exporter.rbs

```diff
@@ -0,0 +1,66 @@
+module Datadog
+  module OpenTelemetry
+    module SDK
+      # Inherits from ::OpenTelemetry::Exporter::OTLP::Metrics::MetricsExporter
+      # but parent class type is not available in RBS
+      class MetricsExporter
+        @telemetry_tags: ::Hash[::String, ::String]
+
+        METRIC_EXPORT_ATTEMPTS: "otel.metrics_export_attempts"
+
+        METRIC_EXPORT_SUCCESSES: "otel.metrics_export_successes"
+
+        METRIC_EXPORT_FAILURES: "otel.metrics_export_failures"
+
+        def initialize: (endpoint: ::String, timeout: ::Float, headers: ::Hash[untyped, untyped], protocol: ::String) -> untyped
+
+        def export: (untyped metrics, ?timeout: ::Float?) -> Integer
+
+        private
+
+        def telemetry: () -> Core::Telemetry::Component?
+      end
+    end
+  end
+end
+
+# External OpenTelemetry types not available in RBS
+module OpenTelemetry
+  def self.meter_provider: () -> untyped
+
+  def self.meter_provider=: (untyped provider) -> void
+
+  module SDK
+    module Resources
+      class Resource
+        def self.create: (::Hash[::String, untyped] attributes) -> untyped
+      end
+    end
+
+    module Metrics
+      class MeterProvider
+        def initialize: (?resource: untyped) -> void
+
+        def add_metric_reader: (untyped reader) -> void
+
+        def shutdown: () -> void
+      end
+
+      module Export
+        class PeriodicMetricReader
+          def initialize: (exporter: untyped, export_interval_millis: ::Integer, export_timeout_millis: ::Integer) -> void
+        end
+      end
+    end
+  end
+
+  module Exporter
+    module OTLP
+      module Metrics
+        class MetricsExporter
+          def initialize: (::String endpoint, ::Float timeout, ::Hash[untyped, untyped] headers) -> void
+        end
+      end
+    end
+  end
+end
```

Comment:
These should go inside the `vendor/` folder -- that's where we place signatures for outside libraries